modules/test/playwright/tests/portal-web/main/html/common/themes/bottom_js.spec.ts

```diff
@@ -108,3 +108,21 @@ test(
 		expect(jsSnippetIIFE).toEqual(true);
 	}
 );
+
+test(
+	'Check URL does not allow XSS injection',
+	{tag: '@LPD-57867'},
+	async ({page}) => {
+		let capturedLogMessage = '';
+
+		page.on('console', (msg) => {
+			if (msg.type() === 'log') {
+				capturedLogMessage = msg.text();
+			}
+		});
+
+		await page.goto('/?snippet=console.log("alert")');
```

Comment:
we can make this test a bit more robust if:

- We emit a random string to the console
- We check console does not include such random string

This way we'll allow any other error/warning/debug message that crosses the console for any other reason when page is loaded